server/src/main/java/com/lunchtime/service/MenuItemDishService.java

```diff
@@ -14,4 +14,7 @@
     Optional<MenuItemDish> findById(Long id);
 
     Page<MenuItemDish> findByRestaurantId(Long id, Pageable pageable);
+
+    Page<MenuItemDish> findMenuItemDishesByRestaurantIdAndDish_CategoryFood_Name(
```

Comment:
Follow Java style guide in method naming
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
https://github.com/Social-projects-Rivne/Lunch_Time/commit/42f8a1365e8252f91ce9643fb8d14a67d6f29d37#diff-045fb6a4100c4cbfc55fe645831ca249R8

Reply:
Thanks. Have forgoten about. Fixed